contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -1221,17 +1221,13 @@ WHEN t8.oid > 0 THEN CAST(t6.relname AS sys.sysname)
 ELSE CAST(SUBSTRING(t6.relname,1,LENGTH(t6.relname)-32-LENGTH(t1.relname)) AS sys.sysname) 
 END AS INDEX_NAME,
 CASE
-WHEN t7.starelid > 0 THEN CAST(0 AS smallint)
```

Comment:
type = 0 is for table statistics which is already handled by first part of union query. And in second part of query it was not required because we were showing info for indexes & primary keys ( clustered and non-clustered indexes respectively).